test/wdio.conf.ts

```diff
@@ -8,12 +8,13 @@ import JsonReporter from './helpers/json-reporter.js';
 import { defaultFunctions as defaultFunctionsInit } from './helpers/default-functions.js';
 import saveScreenshot from './helpers/WDIOMediawikiScreenshotPatch.js';
 import WikibaseApi from './helpers/WDIOWikibaseApiPatch.js';
+import {Options, Frameworks} from "@wdio/types"
```

Comment:
I would have expected to see spaces surrounding this object expansion. Is it your expectation to not have them, and is our linting specific to not having spacing wrap object expansions or agnostic?

Reply:
`eslint` as currently configured doesn't format Typescript files. I had intended to ask if a fix for that should be part of this ticket or a folo?

Reply:
Yes, I do think that would be best as part of this. Are you ok adding that, or does it come with new challenges? Also, I think are we using some sort of WikiMedia standard linting config and if so, we need to see if there is a Typescript specific config we should inherit?

Reply:
So... this has quite a few challenges. There's a project linked below for `eslint-config-wikimedia-typescript`; it hasn't been officially adopted yet, and doesn't seem to work with `wikimedia/selenium`. I copied the `selenium.json` and `server.json` files from eslint-config-wikimedia (linked below), edited them to remove `wikimedia/language/es2018` from the latter, and disabled these rules:

```
       "jsdoc/check-param-names": "off",
        "jsdoc/require-param": "off",
        "no-undef": "off",
        "node/no-missing-import": "off"
```

I'm meeting tomorrow with Michael Große, who worked on the typescript config project, to see if there's a simpler implementation.

https://github.com/wmde/eslint-config-wikimedia-typescript

https://github.com/wikimedia/eslint-config-wikimedia/tree/main

Reply:
Updates: Found an easier way!

Reply:
Nice. Was hoping this would be what came next 🙃